### PR TITLE
Use `trixi_include` from TrixiBase.jl

### PR DESCRIPTION
**Pull request by JoshuaLampert**
Some utility functions moved to the new package TrixiBase.jl.

### CODE REVIEW

**Review by ranocha — APPROVED** [review]
Thanks! This requires your changes to TrixiBase.jl and we're fine, I guess?

**Comment by JoshuaLampert** [comment]
Yes. So let's wait for https://github.com/trixi-framework/TrixiBase.jl/pull/13.

File: Project.toml [hunk 1]
```diff
@@ -34,4 +35,5 @@ SparseArrays = "1"
 StaticArrays = "1"
 SummationByPartsOperators = "0.5.41"
 TimerOutputs = "0.5.7"
+TrixiBase = "0.1"
```
Comment (ranocha) [inline Project.toml:38]:
```suggestion
TrixiBase = "0.1.1"
```
We require the version including your fix for doctests

Comment (JoshuaLampert) [reply Project.toml]: Yes, thanks!

**Review by ranocha — DISMISSED** [review]

**Comment by ranocha** [comment]
It looks like the availability of the new version has not reached all package servers yet. Maybe let's wait a bit and re-trigger CI after a few hours?

**Comment by JoshuaLampert** [comment]
> Maybe let's wait a bit and re-trigger CI after a few hours?

CI is passing now.

**Review by ranocha — APPROVED** [review]
Thanks!

**Pull request merged**